src/workerd/server/BUILD.bazel

```diff
@@ -117,6 +117,7 @@ wd_cc_library(
         ":actor-id-impl",
         ":alarm-scheduler",
         ":workerd_capnp",
+        "//deps/rust:runtime",
```

Comment:
One wrinkle is that every C++ binary that depends on Rust will have to depend on this target. Currently we could just depend on `@workerd//deps/rust:runtime` in `deps/rust/BUILD.lolhtml`, but eventually we'd have to do this for every Rust target which is not clean either. I guess we can do this as-is for now, in the long term we'll have to think something up to avoid confusion

Reply:
I don't think this is a big issue: downstream this library is a no-op, so we have to sprinkle it only in workerd to make things work on win. I'm fine with it.